### ISSUE TITLE
请问作者，对此知乎贴有何回复？

### ISSUE THREAD

**Issue by lck1201** [jfzhang95/PoseAug#5]
RT，偶然看到这个帖子，https://www.zhihu.com/question/459079312
帖子所言属实吗，还是题主断章取义了？

**Comment by Garfield-kh** [comment]
Thank you for the reminder. This should be due to the different experiment settings.

Compared with original VideoPose3D,
1. We use 16 joints configuration instead of using 17 joints, for easier adaptation to the cross-dataset 3DHP with different joint definition (e.g, no nose joint in 3DHP). This would increase MPJPE as the dominator of MPJPE are feet and wrists.
2. We didn’t conduct ‘training and test time flip’ augmentation on H36M dataset for simplicity and avoiding the influence of mixing different augmentation methods. Therefore, the results of MPJPE would be higher than the results reported in the original paper as they use both training and test time flip augmentation.

Compare with original SemGCN,

As there exist four different baseline models and four different 2D keypoints settings (16 experiments in total) to be evaluated, and the experiment settings are different (e.g., training epoch, optimizer, batch size, and learning rate), we train all these models and report the results using a unified training framework with the same hyper-parameter settings for fair comparison. Therefore, the hyper-parameters may be different from the original ones, e.g., SemGCN original setting: batch size 64, learning rate 1e-3; Ours: batch size 1024, learning rate 2e-2.

We verified our unified training framework by comparing its results with the results reported in the original papers. For example,
1. SemGCN-GT: 44.4 (43.8 reported in the original paper https://arxiv.org/pdf/1904.03345.pdf),
2. SimpleBaseline-GT: 43.3 (45.5 reported in the original paper https://arxiv.org/pdf/1705.03098.pdf).

For SemGCN-CPN and VideoPose3D-GT (single-frame), we didn’t find the corresponding results in the original papers. 

Our code will be released within one week, you are welcome to try.

**Comment by JunrQ** [comment]
Amazing work, looking forward to the code, I have a question:

Did you compare the performance of PoseAug with other normal augmentation on your experiment settings? For example, 'training and test time flip’

**Comment by Garfield-kh** [comment]
We evaluated VideoPose3D with both training and test time flip augmentation on H36M and 3DHP datasets.

Under the 2D poses from CPN setting with 16 joints configuration:
The train-test-flip augmentation achieved 53 in H36M, which is similar to ours (52.9), but for the cross-dataset setting, ours (78.4) is much better than its result (85.6).

Under the 2D poses from GT setting with 16 joints configuration:
The train-test-flip augmentation achieved 40.1 in H36M, which is worse than ours (38.2), and for the cross-dataset setting, ours (73) is much better than its result (85).

**Comment by kyang-06** [comment]
Hi all, it's me that created the referred zhihu issue. Apologize for my subjective assumption and have revised the original issue.
As @Garfield-kh clarified, conducting evaluation on 16 joints instead of 17 joints results in similar drop of MPJPE on all methods. So the proposed poseaug shows definitely remarkable margins and a simple but effective augmentation mechanism. 
Say sorry again and thank you for this great work!

**Comment by Garfield-kh** [comment]
@ky66111 Thank you for the updating and please let me know if there is any question.  ^-^
